sys/posix/pthread/include/pthread_cond.h

```diff
@@ -24,6 +24,10 @@
 #   include "msp430_types.h"
 #endif
 
+#if defined(__APPLE__)
```

Comment:
Could you elaborate, i.e., is checking for `__APPLE__` not enough? I haven't seen a case where checking for `__APPLE__` is not enough, do you have a reference when `__MACH__`is required?